docs/assets/css/style.css

```diff
@@ -35,6 +35,18 @@ html {
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box; }
+  
+*::-webkit-scrollbar {
+  width: 20px;
+}
+*::-webkit-scrollbar-track {
+  background: var(--primaryColor);
+}
+*::-webkit-scrollbar-thumb {
+  background-color: white;
+  border-radius: .5em;
+  border: 4px solid var(--primaryColor);
+}
```

Comment:
[Looks like this is something that shouldn't be used](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar)
![image](https://github.com/devicons/devicon/assets/43886029/86f6ae91-c290-4200-a00e-9d00e3cf3f02)

Or perhaps it just needs to be used in conjunction with the firefox equivalent?
https://stackoverflow.com/a/6165489/12206312

More documentation on CSS scrollbar styling: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_scrollbars_styling

Reply:
this is just the vendor specific prefix for all webkit based browsers, and the reason why it does not affect firefox.
only the `border-right: none; ` part affects firefox.

but IMHO the scrollbar looks good on firefox without a change:
![image](https://github.com/devicons/devicon/assets/482423/e279cd6d-eff8-40c3-ac1d-91874f5dfc1a)

(Firefox 118.0.2 on Ubuntu)

Reply:
I guess it's better than it was anyways, so let's just approve this, and improve upon it later if needed :)